cli/cmd/proxy/read/config.go

```diff
@@ -219,9 +219,15 @@ func parseClusters(rawCfg map[string]interface{}, clusterMapping map[string][]st
 			}
 		}
 
+		// Don't display "non-domain name" FQDNs (e.g. local_app)
+		fqdn := cluster.Cluster.FQDN
```

Comment:
Looking at this a bit more, I think FQDNs are not required to have `.`. For example, `localhost` is a valid FQDN. So I think we should skip this change altogether because there won't be a straightforward way to determine whether a string is an FQDN or not

Reply:
I think that makes sense. Envoy refers to `local_app` as an FQDN so I think we can get away with doing the same.